astropy/table/table.py

```diff
@@ -2153,6 +2153,57 @@ def rename_column(self, name, new_name):
 
         self.columns[name].info.name = new_name
 
+    def rename_columns(self, names, new_names):
+        '''
+        Renames multiple columns.
+
+        Parameters
+        ----------
+        names: list
```

Comment:
Since the method accepts `tuple`, say that here and below.  (i.e. `names: list, tuple`, then below `A list or tuple of existing column names`).